src/main.ts

```diff
@@ -28,6 +29,8 @@ main();
 async function main() {
   startConsole(agentIds, messageBus);
 
+  const webSocketServer = new WebSocketServer(messageBus, 8080);
```

Comment:
It doesn't seem like there's a point to making this a class and assigning an instance to a variable, can we just turn it into something like `startWebSocketServer(messageBus, 8080)`?

Reply:
sure - i made a class as I anticipated I might need access to it later on so created a reference